react/src/components/atoms/buttons/ButtonAlert/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './style.css';
+import Icon from '../../icons/Icon';
+
+
+const ButtonAlert = ({
+  text, onClick, hideText, showText
+}) => (
+  <button className="ma__button-alert" onClick={onClick}>
+    <span className="ma__button-alert__show">{showText}</span>
+    <span className="ma__button-alert__hide">{hideText}</span>
+    <Icon />
```

Comment:
Icon has props that it needs. In this case, Icon needs the `name` prop in order to render properly with its generated sprite. If an Icon component is rendered within any component, a single svg sprite is created which will contain a single SVG for each named within your component. The `name` prop is the name of the file you need in the `src/atoms/icons/Icon/assets` directory. In this case, I think you need the chevron icon (chevron.svg), so the name is chevron:
```suggestion
    <Icon name="chevron" />
```